Template/Tag/RaygunTag.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\TagManager\Template\Tag;
+
+use Piwik\Settings\FieldConfig;
+use Piwik\Plugins\TagManager\Template\Tag\BaseTag;
+use Piwik\Validators\NotEmpty;
+
+class RaygunTag extends BaseTag
+{
+    public function getIcon() {
+        return 'plugins/TagManager/images/icons/raygun.svg';
+    }
+
+    public function getParameters() {
+        return array(
+            $this->makeSetting('raygunApiKey', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
+                $field->title = 'Raygun apiKey';
+                $field->uiControl = FieldConfig::UI_CONTROL_TEXT;
+                $field->description = "The DSN (Data Source Name) of your sentry.io project. It should look like 'https://<key>@sentry.io/<project>'";
```

Comment:
should this mention sentry.io?

Reply:
Thanks, I forgot to write the description for it.